ci-operator/step-registry/hypershift/mce/agent/create/hostedcluster/hypershift-mce-agent-create-hostedcluster-commands.sh

```diff
@@ -83,7 +83,6 @@ if [ ! -f "${SHARED_DIR}/id_rsa.pub" ] && [ -f "${CLUSTER_PROFILE_DIR}/ssh-publi
   cp "${CLUSTER_PROFILE_DIR}/ssh-publickey" "${SHARED_DIR}/id_rsa.pub"
 fi
 
-/tmp/${HYPERSHIFT_NAME} --version
```

Comment:
It's better we have this version display in the logs. This will be helpful while debugging

Reply:
For mce 2.7, `/tmp/${HYPERSHIFT_NAME} --version` will be failed, we should use `/tmp/${HYPERSHIFT_NAME} version`

Reply:
Considering this is a common requirement, add the display of the Hypershift version in the mce-install step.